src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -884,7 +879,145 @@ protected void ClearPreparing()
         {
             Interlocked.Exchange(ref _isPreparing, 0);
         }
-
         #endregion
+
+        /// <summary>
+        /// Enable to decode a video data every frame.
+        /// </summary>
+        /// <remarks><para>The player must be in the <see cref="PlayerState.Idle"/> state.
+        /// And, <see cref="DisplayType.None"/> must be set by calling <see cref="SetDisplay"/>.</para>
+        /// <para>A <see cref="VideoFrameDecoded"/> event is called in a separate thread(not in the main loop).</para>
+        /// <para>The audio PCM data can be retrieved using a <see cref="VideoFrameDecoded"/> event as a media packet.
+        /// So if you change the media packet in the <see cref="VideoFrameDecoded"/> event, it will be displayed on the device
+        /// The callback function holds the same buffer that will be drawn on the display device.
+        /// and the <see cref="MediaPacket"/> is available until it's destroyed by <see cref="MediaPacket.Dispose()"/>.
+        /// The packet has to be destroyed as quickly as possible after rendering the data
+        /// and all the packets have to be destroyed before <see cref="Unprepare"/> is called.</para></remarks>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">
+        ///     Operation failed; internal error.
+        ///     -or-<br/>
+        ///     The player is not in the valid state.
+        ///     </exception>
+        /// <seealso cref="PlayerAudioExtractOption"/>
+        /// <seealso cref="DisableExportingVideoFrame"/>
+        /// <since_tizen> 6 </since_tizen>
+        public void EnableExportingVideoFrame()
```

Comment:
It cause behavior break
As is VideoFrameDecoded event occurred without call this API
but to be VideoFrameDecoded event does ont occurred without call this API

So, it cause application regressing issue

Reply:
that's right. i'll roll back video callback and talk about it with others.

Reply:
How about enable callback when only VideoFrameDecoded is not null (listener was existed) at prepare call

``` c#
        private void Prepare()
        {
            if ( _videoFrameDecoded != null) 
            {
                  EnableExportingVideoFrame();
            }
            else 
            {
                DisableExportingVideoFrame();
            }
            NativePlayer.Prepare(Handle).ThrowIfFailed(this, "Failed to prepare the player");
        }
```

or How about introduce a state of ExportVideoFrameState { Enabled, Disabled } and it has default value as `Enabled`

``` c#
 public ExportVideoFrameState ExportVideoFrame { get; set; }
```

Reply:
If so, `VideoFrameDecoded` event should be set in `Idle` state and it's not described in that event.
So, it may also cause application regressing issue.

Reply:
How about second solution?

Reply:
Currently, there's missing description about restriction for `VideoFrameDecoded` event.

Actually, to use `VideoFrameDecoded` event, Display should be set by `null`(PlayerDisplayType.None). If not, event will not be raised. 
(In native function, it's described already and C# TC is already testing it with this scenario.)
So, If user set some window to `Display` property, `VideoFrameDecoded` event will be disabled, otherwise enabled.

@aferin , How about adding missed description in `VideoFrameDecoded` event like native function. 
I think it's not `changing behavior` but `fixing bug - missed restriction`.

Reply:
even if the description is added, there is still a problem as we talked yesterday.
if a user wants to set display none but doesn't want to get callback, how can we support it?

Reply:
In point of view in API usability, I think it's the same with native function usage.
If C# user don't want to get decoded video frame, they should not set event handler to `VideoFrameDecoded` event even if `Display` property is set by `null`.
And Is it meaningful scenario? User don't want to display something and don't want to decoded video frame also, so why user start player?